prov/efa/src/efa_base_ep.c

```diff
@@ -341,3 +341,21 @@ int efa_base_ep_getname(fid_t fid, void *addr, size_t *addrlen)
 	memcpy(addr, &base_ep->src_addr, EFA_EP_ADDR_LEN);
 	return 0;
 }
+
+bool efa_base_ep_support_op_in_order_aligned_128_bytes(struct efa_base_ep *base_ep, enum ibv_wr_opcode op) {
+#if HAVE_EFA_DATA_IN_ORDER_ALIGNED_128_BYTES
+	int ret;
+	switch(op) {
+	case IBV_WR_SEND:
+		break;
+	case IBV_WR_RDMA_WRITE:
+		ret = ibv_query_qp_data_in_order(base_ep->qp, 
```

Comment:
It seems we can simply `return ibv_query_qp_data_in_order`

Reply:
> yep, we can, but would like to keep a pattern for future.

Future code can also use `return`.